### PR TITLE
Issue #1406

### PR DESCRIPTION
**Pull request by shroffk**
Adding the missing help for diirt
Including the diirt repo to the core and applications pom.xml

### CODE REVIEW

**Comment by shroffk** [comment]
@utzeln @berryma4 @willrogers @kasemir 
We have to still remove all references to the old pvmanager.ui to prevent duplicate entries in help

**Comment by berryma4** [comment]
The repository id for diirt is csstudio-thirdparty?

**Comment by shroffk** [comment]
This is a strang commit that came in while doing a rebase

**Comment by shroffk** [comment]
Fixed the incorrect repository listing, thanks for catching that.

**Comment by berryma4** [comment]
I just noticed core/diirt/diirt-repository/pom.xml is also incorrect.  The tar name and the goal for the s3 wagon aren't right.  Could you just use the one that is there, our copy from another core repository (they're all the same for that part).

**Comment by shroffk** [comment]
I used the one from an existing module...I will fix it

**Comment by shroffk** [comment]
I compared the pom with the base/base-repository/pom.xml and the platform/platform-repository/pom.xml They have the same s3 location.

**Comment by berryma4** [comment]
sorry I wasn't very clear:
```xml
<plugin>
            <groupId>org.eclipse.tycho</groupId>
            <artifactId>tycho-p2-repository-plugin</artifactId>
            <version>${tycho.version}</version>
            <configuration>
              <finalName>cs-studio-core-${project.parent.artifactId}-${project.version}</finalName>
              <repositoryName>cs-studio-core-${project.parent.artifactId}</repositoryName>
              <includeAllDependencies>true</includeAllDependencies>
            </configuration>
```

File: core/diirt/diirt-repository/pom.xml [hunk 1]
```diff
@@ -162,3 +162,167 @@
...
+                                    <destination>${csstudio.local.repo}</destination>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.eclipse.tycho</groupId>
+                        <artifactId>tycho-p2-repository-plugin</artifactId>
+                        <version>${tycho.version}</version>
+                        <configuration>
```
Comment (berryma4) [inline core/diirt/diirt-repository/pom.xml]:
```xml
<finalName>cs-studio-core-${project.parent.artifactId}-${project.version}</finalName>
              <repositoryName>cs-studio-core-${project.parent.artifactId}</repositoryName>
              <includeAllDependencies>true</includeAllDependencies>
```

File: core/diirt/diirt-repository/pom.xml [hunk 2]
```diff
@@ -162,3 +162,167 @@
...
+                            <destination>${project.build.directory}/centralrepository</destination>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.eclipse.tycho</groupId>
+                <artifactId>tycho-p2-repository-plugin</artifactId>
+                <version>${tycho.version}</version>
+                <configuration>
```
Comment (berryma4) [inline core/diirt/diirt-repository/pom.xml]:
```xml
<finalName>cs-studio-core-${project.parent.artifactId}-${project.version}</finalName>
          <repositoryName>cs-studio-core-${project.parent.artifactId}</repositoryName>
```

**Pull request merged**